package.json

```diff
@@ -13,22 +13,21 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "node_modules/.bin/tsc",
-    "test": "mocha --require ./node_modules/ts-node/register --recursive test/**/*.spec.ts",
+    "test": "NODE_OPTIONS=--trace-deprecation mocha --require ./node_modules/ts-node/register --recursive test/**/*.spec.ts",
     "test:debug": "mocha --inspect-brk --require ./node_modules/ts-node/register --recursive test/**/*.spec.ts",
     "coverage": "node_modules/.bin/nyc npm run test",
     "publish:coverage": "node_modules/.bin/nyc report --reporter=text-lcov > coverage.lcov && curl -s https://codecov.io/bash | bash"
   },
   "dependencies": {
     "mocha": "^5.2.0",
-    "nock": "^10.0.1",
+    "nock": "^13",
```

Comment:
why not specify the minor + patch versions as well? Latest is `13.3.0` as of writing this.

Reply:
Addressed in 44a2b4d0aea35b5c8fdbe2db50fcb1d466f464eb.